src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -58,6 +58,28 @@ function warnIfValueIsNull(props) {
   }
 }
 
+function warnIfDuplicateValues(inst) {
```

Comment:
I noticed `ReactDOMNullInputValuePropDevtool.js` exports `ReactDOMUnknownPropertyDevtool`. Is that correct or a copy/paste error?